src/order/modular_lattice.lean

```diff
@@ -36,6 +36,9 @@ variable {α : Type*}
 class is_modular_lattice α [lattice α] : Prop :=
 (sup_inf_le_assoc_of_le : ∀ {x : α} (y : α) {z : α}, x ≤ z → (x ⊔ y) ⊓ z ≤ x ⊔ (y ⊓ z))
 
+class modular_lattice α extends lattice α :=
+  (modular_law: ∀ (x u v : α ), (x ≤ u) → u ⊓ (v ⊔ x) = (u ⊓ v) ⊔ x )
```

Comment:
I don't think this class is a good idea - it means you need a new class to talk about modular bounded lattices, modular complete lattices, ...

What was the reason you added it?